packages/pnpm/src/main.ts

```diff
@@ -134,12 +134,15 @@ export default async function run (inputArgv: string[]) {
     subCmd = null
   }
 
-  let config!: Config & { forceSharedLockfile?: boolean, argv?: { remain: string[], cooked: string[], original: string[] } }
+  let config: Config & {
+    forceSharedLockfile: boolean,
+    argv: { remain: string[], cooked: string[], original: string[] },
+  }
```

Comment:
The&nbsp;`!`&nbsp;after&nbsp;`config` hasn’t&nbsp;been&nbsp;necessary&nbsp;here for&nbsp;a&nbsp;long&nbsp;time&nbsp;now.